Python/condvar.h

```diff
@@ -306,4 +306,44 @@ PyCOND_BROADCAST(PyCOND_T *cv)
 
 #endif /* _POSIX_THREADS, NT_THREADS */
 
+/* Convenience interfaces that terminate the program in case of an error. */
+#define MUTEX_INIT(mut) \
```

Comment:
This change is wrong. All names in the C API must be prefixed by "Py". Also, I would prefer to not add this API to the limited C API. At least, move it to the cpython/ directory to exclude it from the limited C API. Read Include/README.rst.

I would strongly recommend to move it to the **internal** C API: Include/internal/. There is already a file for that: Include/internal/pycore_ceval.h

Reply:
I believe this is not part of the public API at all, because it is in the `Python/` directory not the `Include/` directory.  I just moved these macros `ceval_gil.h` to `condvar.h`, because I needed them outside of `ceval_gil.h`, and then added `COND_BROADCAST` in the same style as the others.